src/status_im/protocol/handlers.cljs

```diff
@@ -254,7 +254,7 @@
 (re-frame/reg-fx
  ::mark-trusted-peer
  (fn [{:keys [wnode web3]}]
-   (inbox/mark-trusted-peer wnode
+   (inbox/mark-trusted-peer web3 wnode
```

Comment:
i suggest to format arguments in a column as it's done below in `inbox/request-messages` call